tensorflow/lite/micro/debug_log.h

```diff
@@ -17,19 +17,23 @@ limitations under the License.
 
 #ifdef __cplusplus
 #include <cstdarg>
+#include <cstddef>
 #else
 #include <stdarg.h>
+#include <stddef.h>
 #endif  // __cplusplus
 
 #ifdef __cplusplus
 extern "C" {
 #endif  // __cplusplus
 
-// This function should be implemented by each target platform, and provide a
+// These functions should be implemented by each target platform, and provide a
 // way for strings to be output to some text stream. For more information, see
-// the tensorflow/lite/micro/debug_log.cc file.  This function should support
+// the tensorflow/lite/micro/debug_log.cc file.  These functions should support
 // standard C/C++ stdio style formatting operations.
 void DebugLog(const char* format, va_list args);
+int DebugLogVsnprintf(char* buffer, size_t buf_size, const char* format,
```

Comment:
Nit: I'd suggest we call this DebugVsnprintf. It's not really doing any logging.

Reply:
changed